web/src/features/fba/components/infoPanel/AdvisoryText.tsx

```diff
@@ -232,14 +293,24 @@ const AdvisoryText = ({
             {getHighProportionFuelsString()}
           </Typography>
         )}
-        {!isUndefined(zoneStatus) && !isUndefined(minWindSpeeds) && (
-          <Typography sx={{ whiteSpace: 'pre-line' }} data-testid="advisory-message-min-wind-speeds">
-            {minWindSpeeds}
+        {!isUndefined(zoneStatus) && earlyOrLowWindText && (
```

Comment:
There are a lot of repetitive checks for `!isUndefined(zoneStatus)`. Can this be simplified?

Reply:
Thank you!